### DISCUSSION TITLE
Bypass

### DISCUSSION THREAD

**Discussion by ian4u** [DAMcraft/MeteorServerSeeker#32] [Ideas]
Can you add a friendlist or bypass for servers you not trying to scan. That would be awsome 
Also love the addon <3

**Comment by rewrite** [comment]
What?

**Reply by ian4u** [reply]
A whitelist for servers you dont wanna scan

**Reply by rewrite** [reply]
The scanning is not client-side

**Reply by ian4u** [reply]
Than just make it so that the whitlisted servers wont show up ?

**Reply by rewrite** [reply]
Why would you want to to this? On default settings, the chance of a server being in the search results is >1:100.000

**Reply by ian4u** [reply]
Well than im the luckiest guy ever bc i did it

**Comment by lina-x64** [comment]
I suppose if your query is specific this might happen, I will try to see how I solve this!
